### PR TITLE
New Worker Panel Styles

### PR DESCRIPTION
**Pull request by KPrasch**
- Based on #75
- Styles worker address collection closer to the spec
- Needs follow-up (see <https://github.com/nucypher/nuit/issues/79>)

Screenshot from 2021-05-05 03-44-46

Screenshot from 2021-05-05 03-42-36

### CODE REVIEW

File: packages/react-app/src/components/chart.js [hunk 1]
```diff
@@ -109,18 +69,58 @@ function scrubPeriods(periods) {
     return formattedPeriods
 }
 
-function calculateNewStakerDelta(period1, period2) {
+//
+// Components
+//
 
+class StyledActiveDot extends React.Component {
```
Comment (vepkenez) [inline packages/react-app/src/components/chart.js:76]: why do this as a class based component?

File: packages/react-app/src/components/chart.js [hunk 2]
```diff
@@ -23,60 +22,17 @@ function dateToUTCEpoch(d) {
     return Number(Math.round(utcMilllisecondsSinceEpoch / 1000))
 }
 
-function oneYearAgo() {
+function calculatePastEpoch(months_ago) {
```
Comment (vepkenez) [inline packages/react-app/src/components/chart.js:25]: maybe functions like this should go in here?
https://github.com/nucypher/nuit/blob/main/packages/react-app/src/services/nucypher.js

**Comment by vepkenez** [comment]
Something seems wrong with this PR.  What are all these commits?  Most of them seem to have nothing to do with styling this one little panel.

**Comment by KPrasch** [comment]
> Something seems wrong with this PR.  What are all these commits?  Most of them seem to have nothing to do with styling this one little panel.


This PR is based over #75, so with that merged it needs a rebase for GH to properly show the history (I'm hoping Microsoft will support automatically resolving it eventually).

**Comment by vepkenez** [comment]
so safe to merge then... ok

**Pull request merged**